src/index.html

```diff
@@ -11,7 +11,98 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>HTML Form</h1>
+    <form action="https://mate-academy-form-lesson.herokuapp.com/create-application" method="post">
+      <fieldset class="fieldset">
+        <legend>Personal information</legend>
+        <label class="form__field">Surname:
+          <input type="text" 
+          autocomplete="off" 
+          name="surname"
+          >
```

Comment:
```suggestion
          <input 
            type="text" 
            autocomplete="off" 
            name="surname"
          >
```
3 point in checklist, make sure to fix it everywhere not just here